extensions/wikia/Forum/ForumHelper.class.php

```diff
@@ -1,23 +1,77 @@
 <?
+
 class ForumHelper {
-	public static $forumNamespaces = [NS_WIKIA_FORUM_BOARD, NS_WIKIA_FORUM_TOPIC_BOARD, NS_WIKIA_FORUM_BOARD_THREAD];
+	public static $forumNamespaces = [
+		NS_WIKIA_FORUM_BOARD,
+		NS_WIKIA_FORUM_TOPIC_BOARD,
+		NS_WIKIA_FORUM_BOARD_THREAD,
+	];
 
 	public static function isForum() {
 		global $wgTitle;
 		if ( $wgTitle instanceof Title ) {
-			if ( $wgTitle->isSpecial( 'Forum' ) || in_array( $wgTitle->getNamespace(), self::$forumNamespaces ) ) {
+			if ( $wgTitle->isSpecial( 'Forum' ) ||
+			     in_array( $wgTitle->getNamespace(), self::$forumNamespaces )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)